### PR TITLE
Add redis prefix to be able save per-instance statistics

### PR DESCRIPTION
**Pull request by InboxViktorV**

### CODE REVIEW

File: DependencyInjection/Compiler/ResolveAdapterDefinitionPass.php [hunk 1]
```diff
@@ -31,7 +31,16 @@ public function process(ContainerBuilder $container)
         $definition->setAbstract(false);
         $definition->setClass($adapterClasses[$container->getParameter('prometheus_metrics_bundle.type')]);
         if ('redis' === $container->getParameter('prometheus_metrics_bundle.type')) {
-            $definition->setArguments([$container->getParameter('prometheus_metrics_bundle.redis')]);
+            $redisArguments = $container->getParameter('prometheus_metrics_bundle.redis');
+
+            $prefix = $redisArguments['prefix'] ?? null;
+            if ($prefix) {
```
Comment (denisvmedia) [inline DependencyInjection/Compiler/ResolveAdapterDefinitionPass.php:37]: I think, we should have here a strict comparison, or maybe `!empty($prefix)` call, to be a little be more type-safe.
P.S. Would you please also remove all those new lines as here I don't think they give any semantical sense.

File: DependencyInjection/Configuration.php [hunk 2]
```diff
@@ -65,6 +65,7 @@ public function getConfigTreeBuilder(): TreeBuilder
                         ->booleanNode('persistent_connections')->end()
                         ->scalarNode('password')->end()
                         ->integerNode('database')->end()
+                        ->scalarNode('prefix')->end()
```
Comment (denisvmedia) [inline DependencyInjection/Configuration.php:68]: Do we need any validation here? I'm not sure if the prefix can be anything.

**Comment by denisvmedia** [comment]
Would you please also change the destination branch to `develop`.

**Comment by denisvmedia** [comment]
@InboxViktorV Thank for your efforts. I will merge your PR right now. It will be released soon.

**Comment by InboxViktorV** [comment]
@denisvmedia Cool, thanks;)

**Pull request merged**